src/admin/content/content.service.ts

```diff
@@ -30,11 +30,16 @@ export const fetchContentItemById = async (id: number): Promise<Avo.Content.Cont
 	}
 };
 
-export const fetchContentItems = async (limit: number): Promise<Avo.Content.Content[] | null> => {
+export const fetchContentItems = async (
+	keyword: string,
+	limit: number
+): Promise<Avo.Content.Content[] | null> => {
 	try {
+		const where = { title: { _ilike: keyword } };
+
 		const response = await dataService.query({
 			query: GET_CONTENT_PAGES,
-			variables: { limit, order: { title: 'asc' } },
+			variables: { where, limit, order: { title: 'asc' } },
```

Comment:
door de query op deze manier te schrijven sta je toe dat een ingelogde gebruiker gelijk welke where props meegeeft. Dit is op zich geen probleem.

Als je strenger wil zijn kan je een query schrijven: GET_CONTENT_PAGES_BY_TITLE en dan wordt enkel die query gewhitelist in graphql.

Ik wijs met deze comment enkel op de afweging die je moet maken mbt die whitelisting behavior, je mag zelf beslissen of het strenger moet of niet.